src/CreateOverlay.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import OverlayMixin from './OverlayMixin';
+import RootCloseWrapper from './RootCloseWrapper';
+
+
+export default function createOverlayedComponent (render, { renderChildren = false, rootClose = true } = {}) {
```

Comment:
no its not needed, I have it deleted downstream here I just haven't add a chance to push it